### DISCUSSION TITLE
Few Questions around seasonality and holidays

### DISCUSSION THREAD

**Discussion by dipranjan** [ourownstory/neural_prophet#1184] [Q&A - get help using NeuralProphet] [answered]
1. I am working on my company data which is on fiscal year - Feb to March and runs in quarterly cycles. The time series has weekly data as in only 13 datapoints (each saturday) for a quarter. What is the suggested best practice to include this fiscal year and quarterly cycles as a part of seasonality.

2. Every year just before the first saturday of the year there is a dip for 2 weeks and then spike for 1 week, post which it stabilizes. Given I have the dates when the spike is expected I am not able to model the dip and peaks even by adding it as a holiday. Any suggestion on what to do here? 
Last question on this topic given weekly data does lower_window of -1 indicate one step in time or 1 day period?

Apologies for the long question.

**Comment by judussoari** [comment, marked as answer]
1. Check out conditional seasonality, which is a feature we have implemented recently. Use m.add_seasonality() for this
2. I think this is already well captured with yearly seasonality and nonlinear autoregression, as these are yearly repeating patterns
3. Should be timesteps, so in your case a week
